scipy/ndimage/measurements.py

```diff
@@ -1205,6 +1205,33 @@ def maximum_position(input, labels=None, index=None):
     --------
     label, minimum, median, maximum_position, extrema, sum, mean, variance,
     standard_deviation
+    
+    Examples
+    --------
+    >>> from scipy import ndimage
+    >>> import numpy as np
+    >>> a = np.array([[1, 2, 0, 0],
+    ...               [5, 3, 0, 4],
+    ...               [0, 0, 0, 7],
+    ...               [9, 3, 0, 0]])
+    >>> ndimage.maximum_position(a)
+    (3, 0)
+
+    Features to process can be specified using `labels` and `index`:
+
+    >>> lbl = np.matlib.repmat([0,1,2,3],4,1)
```

Comment:
I'd prefer an explicit use of `np.array` here, like a few lines above. Clearer to the reader, and `np.matlib.repmat` is a function that's best avoided.